src/modules/back-to-top/back-to-top.component.html

```diff
@@ -0,0 +1,3 @@
+<div #backToTop class="back-to-top" (click)="goToTop()" [ngStyle]="{'background-color': backgroundColor, 'color': color}">
```

Comment:
My rationale for this is to allow the component to be used elsewhere. If a site has a specific design style, you wouldn't want the default color to clash.